io_context.go

```diff
@@ -108,12 +108,16 @@ func AllocIOContext(bufferSize int, writable bool, readFunc IOContextReadFunc, s
 	return
 }
 
-// https://ffmpeg.org/doxygen/7.0/avio_8c.html#ab1b99c5b70aa59f15ab7cd4cbb40381e
-func OpenIOContext(filename string, flags IOContextFlags) (*IOContext, error) {
+// https://ffmpeg.org/doxygen/7.0/avio_8c.html#ae8589aae955d16ca228b6b9d66ced33d
+func OpenIOContext(filename string, flags IOContextFlags, d *Dictionary) (*IOContext, error) {
```

Comment:
Can you also add `ii *IOInterrupter` as an argument and provide it to `avio_iopen2`?
```go
var cii *C.AVIOInterruptCB = nil
if ii != nil {
  cii = &ii.c
}
C.avio_open2(&c, cfi, C.int(flags), cii, dc)
```
Don't forget to update examples and other places where it's used 👍